samza-azure/src/main/java/org/apache/samza/system/eventhub/consumer/EventHubSystemConsumer.java

```diff
@@ -254,6 +256,13 @@ private EventHubClientManager createOrGetEventHubClientManagerForSSP(String stre
     return eventHubClientManager;
   }
 
+  private void boundedInitializeEventHubsManagers() {
```

Comment:
Could you also add this logic to the producer? It should may encounter the same problem. https://github.com/apache/samza/blob/a702b3bb4a159804a2f7234710231b0dbf570b90/samza-azure/src/main/java/org/apache/samza/system/eventhub/producer/EventHubSystemProducer.java#L167

Reply:
done